### PR TITLE
[FIX] stock: Put correctly stock.move and stock.quant dates

### PR DESCRIPTION
**Pull request by pedrobaeza**
As the standard code overwrites date field when transferring, we need to keep the old date, and the created quants also need to take this date instead the date when the migration is done.

With this last patch, I think we can consider the stock migration as stable, having migrated already 4 DBs with plenty of stock stuff.

### CODE REVIEW

File: openerp/openupgrade/doc/source/modules70-80.rst [hunk 1]
```diff
@@ -416,7 +416,7 @@ Status :
 +-----------------------------------+-----------------------------------+
 |share                              | Nothing to do                     |
 +-----------------------------------+-----------------------------------+
-|stock                              | Done (experimental)               |
+|stock                              | Done                              |
```
Comment (StefanRijnhart) [inline openerp/openupgrade/doc/source/modules70-80.rst:419]: Any experiences with multi-company + multi-warehouse? That was the aspect that I was unsure of when I added my bit to the stock migration script.

**Comment by StefanRijnhart** [comment]
:+1: overall, thanks!

File: openerp/openupgrade/doc/source/modules70-80.rst (hunk 1 above)
Comment (pedrobaeza) [inline openerp/openupgrade/doc/source/modules70-80.rst:419]: Uhm, not for multi-company, but with multi-warehouse I didn't have any special problem

Comment (StefanRijnhart) [inline openerp/openupgrade/doc/source/modules70-80.rst:419]: OK, thanks!

**Comment by legalsylvain** [comment]
thanks !
:+1:

**Pull request merged**